Sources/BraintreePayPalMessaging/BTPayPalMessagingClient.swift

```diff
@@ -0,0 +1,150 @@
+import UIKit
+import SwiftUI
+import PayPalMessages
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Use `BTPayPalMessagingClient` to display PayPal messages to promote offers such as Pay Later and PayPal Credit to customers.
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public class BTPayPalMessagingClient: UIView {
+
+    // MARK: - Properties
+
+    public weak var delegate: BTPayPalMessagingDelegate?
+
+    var apiClient: BTAPIClient
+
+    // MARK: - Initializers
+
+    ///  Initializes a PayPal Messaging client.
+    /// - Parameter apiClient: The Braintree API client
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+
+        super.init(frame: .zero)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Method
+
+    /// Creates a `BTPayPalMessagingClient` to be displayed to promote offers such as Pay Later and PayPal Credit to customers.
+    /// - Parameter request: an optional `BTPayPalMessagingRequest`
+    public func createView(_ request: BTPayPalMessagingRequest = BTPayPalMessagingRequest()) {
+        apiClient.fetchOrReturnRemoteConfiguration { configuration, error in
+            if let error {
+                self.delegate?.onError(self, error: error)
+                return
+            }
+
+            guard let configuration else {
+                self.delegate?.onError(self, error: BTPayPalMessagingError.fetchConfigurationFailed)
+                return
+            }
+
+            guard let clientID = configuration.json?["paypal"]["clientId"].asString() else {
+                self.delegate?.onError(self, error: BTPayPalMessagingError.payPalClientIDNotFound)
+                return
+            }
+
+            let messageData = PayPalMessageData(
+                clientID: clientID,
+                environment: configuration.environment == "production" ? .live : .sandbox,
+                amount: request.amount,
+                placement: request.placement?.placementRawValue,
+                offerType: request.offerType?.offerTypeRawValue
+            )
+
+            messageData.buyerCountry = request.buyerCountry
+
+            let messageConfig = PayPalMessageConfig(
+                data: messageData,
+                style: PayPalMessageStyle(
```

Comment:
Question, and this may be something we bring to PP or product folks to get confirmation.

Do we need to be concerned about merchants opting logo types and colors that are NOT meant to be together, per our brand guidelines? 

Alternatively, are there logo and color combinations that absolutely would not pass an accessibility check?

Asking because I'm curious if we may want to surface a developer error for these edge case situations.

Reply:
> Do we need to be concerned about merchants opting logo types and colors that are NOT meant to be together, per our brand guidelines?

Yep the colors are black, white, monochrome, and greyscale and all already approved to go with the logos currently.

> Alternatively, are there logo and color combinations that absolutely would not pass an accessibility check?

PayPal Messages if working with the accessibility team on accessibility across the board. Imo, if there are "bad" combos the messaging SDK should surface as an error instead of our SDK.

Reply:
> Imo, if there are "bad" combos the messaging SDK should surface as an error instead of our SDK.

Agreed, it's always been our practice to be messengers and let the API/Payment Method/downstream component to be the source of truth on validation decisions. Let's check in with our friends in PayPal Messaging about this consideration and move on.